web/config/environment.js

```diff
@@ -45,6 +45,7 @@ module.exports = function (environment) {
       appID: getEnv("ALGOLIA_APP_ID"),
       docsIndexName: getEnv("ALGOLIA_DOCS_INDEX_NAME", "docs"),
       draftsIndexName: getEnv("ALGOLIA_DRAFTS_INDEX_NAME", "drafts"),
+      templateIndexName: getEnv("ALGOLIA_TEMPLATE_INDEX_NAME","template"),
```

Comment:
have you added an entry in `config.hcl` to fetch `"ALGOLIA_TEMPLATE_INDEX_NAME` or added any relevant environmental variable, and what is the default for it?